obelisk_ws/src/obelisk_msgs/obelisk_sensor_msgs/msg/ObkImu.msg

```diff
@@ -0,0 +1,16 @@
+# Obelisk compliant message type to hold IMU information
+
+string MESSAGE_NAME="ObkImu"
+
+std_msgs/Header header
+
+geometry_msgs/Quaternion orientation
+float64[9] orientation_covariance # Row major about x, y, z axes
```

Comment:
I don't feel like this "makes sense" in the sense of differential geometry.

I think if you're going to define the orientation noise using something like a Gaussian, it should be over something like the elements of the Lie algebra se(3), and noise elements should have the exponential map applied to them before being "added" (in the sense of composing two rotations) back to the original rotation.

The main downside to this is that it makes the definition of the noise (in the tangent space) a bit less intuitive to grok, but the main upside is that it's a bit more mathematically coherent. Also, if the noise is small, elements of the tangent space are very similar to the elements of the actual group (in the ambient space).